apps/schedulii-ui/src/pages/Calendar.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  startOfMonth,
+  endOfMonth,
+  eachDayOfInterval,
+  format,
+  subDays,
+  addDays,
+  subMonths,
+  addMonths,
+  isSameDay,
+} from 'date-fns';
+
+import { useState } from 'react';
+
+const Calendar = () => {
+  // Get details about the current month
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const daysOfWeek = [
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday',
+  ];
+  const firstDayOfMonth = startOfMonth(currentDate);
+  const lastDayOfMonth = endOfMonth(currentDate);
+  const diffStart = daysOfWeek.indexOf(format(firstDayOfMonth, 'iiii')); // returns how many days before current month to fill in grid
+  const diffEnd = 6 - daysOfWeek.indexOf(format(lastDayOfMonth, 'iiii')); // returns how many days after current month to fill in grid
+
+  // Generate array of dates for the current month
+  const datesInMonth = eachDayOfInterval({
+    start: firstDayOfMonth,
+    end: lastDayOfMonth,
+  });
+
+  const datesInLastMonth = eachDayOfInterval({
+    start: subDays(startOfMonth(currentDate), diffStart),
+    end: subDays(startOfMonth(currentDate), 1),
+  });
+
+  const datesInNextMonth = eachDayOfInterval({
+    start: addDays(endOfMonth(currentDate), 1),
+    end: addDays(endOfMonth(currentDate), diffEnd),
+  });
+
+  // Classname var
+  const otherMonths = 'text-xl text-slate-400';
+  const buttons = 'mx-5 p-2 border-4';
+
+  // Render calendar
+  return (
+    <>
+      <div>
+        <h1 className="text-3xl">Calendar</h1>
+        <div className="flex">
+          <button
+            className={buttons}
+            onClick={() => setCurrentDate(subMonths(currentDate, 1))}
+          >
+            {' '}
+            Left Button{' '}
```

Comment:
Probably just emojis or `<` is enough. We don't need the text since it blows up the button size.